src/test/java/com/google/api/codegen/testsrc/libraryproto/library.proto

```diff
@@ -423,6 +423,10 @@ message PublishSeriesRequest {
 
   // Uniquely identifies the series to the publishing house.
   SeriesUuid series_uuid = 5;
+
+  // TODO(pongad): Come up with a reason for this path to be here.
```

Comment:
Yes, please do. This does not provide a clear mental model. :(

Maybe the `parent` in an `AddCommentsRequest` could be a `Book`?